README.rst

```diff
@@ -280,8 +280,27 @@ Please read the Documentation's
 `Contributing <http://rauc.readthedocs.io/en/latest/contributing.html>`_
 section for more details.
 
+License
+-------
+
+Copyright (C) 2015–2023 RAUC project
+
+RAUC is free software; you can redistribute it and/or modify it under the terms
+of the GNU Lesser General Public License as published by the Free Software
+Foundation; either version 2.1 of the License, or (at your option) any later
+version.
+
+This library is distributed in the hope that it will be useful, but WITHOUT ANY
+WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+PARTICULAR PURPOSE. See the GNU Lesser General Public License for more details.
+
+You should have received a copy of the GNU Lesser General Public License along
+with this source code, see the file named `COPYING`_. If not, see
+<https://www.gnu.org/licenses/>.
```

Comment:
This does not seem to be a valid link. Either write it as plain text or use \`text ¸\<link\>_\` syntax.

Reply:
It works on https://github.com/rohieb/rauc/tree/license-clarification#license, though.

Reply:
@ejoerns Probably not. Other URLs in the README are plain text as well.

Reply:
It had angle brackets in the source, and I still had Markdown syntax in mind. Fixed.